### PR TITLE
Change ssd to lsd in README example

### PR DESCRIPTION
**Pull request by wadehammes**
## Description

Update README to fix miscalled `ssd` to `lsd`.

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by wadehammes** [comment]
@lirantal I think this is why I wasn't getting my token from localStorage in #36, the example in the README had a typo for the lsd portion.

**Comment by wadehammes** [comment]
@lirantal localStorage isn't persisting between tests using this approach, however utilizing the Cypress localStorage plugin (https://www.npmjs.com/package/cypress-localstorage-commands) works wonderfully:

```
import 'cypress-localstorage-commands';

before(() => {
  describe('SSO Login', () => {
    const username = Cypress.env('googleUser');
    const password = Cypress.env('googlePass');
    const loginUrl = Cypress.env('loginUrl');
    const cookieName = Cypress.env('cookie') // "jwt";
    const socialLoginOptions = {
      username,
      password,
      loginUrl,
      headless: true,
      logs: false,
      loginSelector: '.abcRioButton',
      postLoginSelector: '#searchWrapper',
      isPopup: true,
      popupDelay: 1000,
      args: ['--no-sandbox', '--disable-setuid-sandbox'],
    };

    cy.clearLocalStorageSnapshot();

    return cy.task('GoogleSocialLogin', socialLoginOptions).then(({lsd}) => {
      const hasJwt = Object.keys(lsd)
        .filter((item) => item === cookieName)
        .pop();

      if (hasJwt) {
        cy.window().then(() => {
          Object.keys(lsd).forEach((key) => {
            cy.setLocalStorage(key, lsd[key]);
          });
        });

        cy.saveLocalStorage();
      }
    });
  });
});

// Check for initial results on every test
beforeEach(() => {
  // restores localstorage from initial login
  cy.restoreLocalStorage();

  cy.server();
  cy.route('GET', 'search/*').as('getInitialResults');
  cy.visit('/');

  cy.wait('@getInitialResults').then((response) => {
    assert.isArray(response.response.body.results);
    assert.isNotNull(response.response.body.results);
  });
});

// Save localStorage after all tests
afterEach(() => {
  cy.saveLocalStorage();
});
```

Want me to add an addendum to the README with this plugin information and how to use in an example (I can clean up the above to remove unnecessary lines)?

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ return cy.task('GoogleSocialLogin', socialLoginOptions).then(({cookies,lsd,ssd})
 
   cy.window().then(window => {
       Object.keys(ssd).forEach(key => window.sessionStorage.setItem(key, ssd[key]));
-      Object.keys(lsd).forEach(key => window.sessionStorage.setItem(key, ssd[key]));
+      Object.keys(lsd).forEach(key => window.localStorage.setItem(key, lsd[key]));
```
Comment (lirantal) [inline README.md:66]: oh wow, great catch! 
as my friend @shaninja says: 🦅👁 ;-)

**Review by lirantal — APPROVED** [review]

**Comment by lirantal** [comment]
@wadehammes as for the comment on https://github.com/lirantal/cypress-social-logins/pull/45#issuecomment-652992195 - sure I'd love it if you wanted to send in another PR to update the README with that doc entry, thank you for proposing! 🤗❤️

**Pull request merged**